terraform/main.tf

```diff
@@ -0,0 +1,28 @@
+resource "juju_application" "webui" {
+  name  = "webui"
+  model = var.model_name
+
+  charm {
+    name    = "sdcore-webui-k8s"
+    channel = var.channel
+  }
+
+  units = 1
+  trust = true
+}
+
+resource "juju_integration" "webui-db" {
```

Comment:
Should relations be part of the `bundle` terraform module?  I'm not sure how I feel about them being part of the charm module

Reply:
It's also possible to set all the relations in the bundles. But I only defined the mandatory relations in the charms' TF modules. I  think there some advantages of it:
- We can run the integration tests for charms using their TF modules easily. All the requirements in order to deploy the charm is already included there.
- If we define all the relations in the higher level (bundles), that module will be very complex and not easy to maintain.